app/gui/src/controller/searcher/component/builder.rs

```diff
@@ -316,6 +320,17 @@ impl List {
     }
 }
 
+fn component_is_private(component: &Component) -> bool {
+    match &component.data {
+        component::Data::FromDatabase { entry, .. } =>
+            entry.documentation.iter().any(|doc| match doc {
+                DocSection::Tag { name, .. } => name == "PRIVATE",
```

Comment:
extrct that as const. We are extracting all literals as consts. Plus, check if itdoesnt exists anywhere in the app. Put it in a module with language construct definitions pls.

Reply:
> extrct that as const. We are extracting all literals as consts. Plus, check if itdoesnt exists anywhere in the app. Put it in a module with language construct definitions pls.

@wdanilo Are you aware that this is not a language construct, but the documentation tag? I'd put it in the documentation-related crate instead. Or just here.

Reply:
Documentation is IMO part of language specification. It affects how these things are also accessible from code. IMO this is language-binding thing.

Reply:
> Documentation is IMO part of language specification. It affects how these things are also accessible from code. > IMO this is language-binding thing. 

This is not true AFAIK. You can happily access those methods from the code. The tag just hides them in the Component Browser.

Reply:
Yes, you can for now. But we want to introduce a mode where you could not from external modules if you are not enabling "allow unsafe access" or something like that in the language. It is not supported in Engine yet, but it is part of the language.